### DISCUSSION TITLE
How do you make the TV have a bit more weight?

### DISCUSSION THREAD

**Discussion by miguelarios** [Hankanman/Area-Occupancy-Detection#11] [Q&A] [answered]
Just curious how is the TV or other devices affect or weigh into the occupancy?

**Comment by Hankanman** [comment]
The weights are currently pre-determined, next release will let you configure them, currently a media device has a high weight but not as high as motion. For now you can perhaps lower the threshold instead

**Reply by Hankanman** [reply, marked as answer]
Release is out! You should now be able to change the weight in the config
